catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -22,13 +22,18 @@
             type="String" default=""/>
 
         <AD description="Optional: Copy the ingested files into a backup directory under the monitored directory named /.ingested  -  NOTE: this will double the amount of disk space required for ingested files in this monitored directory if its Processing Directive includes storing the file in the DDF Content Repository."
-            name="Copy Files to Backup Directory" id="copyIngestedFiles" required="false"
-            type="Boolean" default="false"/>
+            name="Processing Mechanism" id="processingMechanism" required="false"
```

Comment:
Thanks for reminding me. I will update this, though I expect someone will have further feedback on the new verbiage. 😄